trading-rules.adoc

```diff
@@ -4,28 +4,59 @@
 
 _This document is under development. Subscribe to https://github.com/bisq-network/bisq-docs/issues/80[bisq-network/bisq-docs#80] for updates._
 
-To trade on Bisq, you assume one of two roles. You either make a new offer to buy or sell an asset (offer maker), or you agree to accept an existing offer to buy or sell an asset (offer taker).
+To trade on Bisq, you assume one of two roles. You either buy (BTC buyer) or sell (BTC seller) bitcoin.
 
 This document covers the rules and expectations for each role, as well as details on the process for arbitrating disputes.
 
-== Offer maker
+== BTC buyer
 
-_Coming soon._
+* Must make payment from the account used to make/take the offer. The identifiers must match.
+* Must enter the correct reference text in the payment transfer.
+* Must click "Payment started" after having done so.
+* Is responsible for providing the arbitrator with tamper proof evidence of their payment in case of dispute.
+* Must be available, in case of dispute, 48 hours after the max. trade period runs out.
+* Has 48 hours to respond to each arbitrator inquiry.
 
-== Offer taker
+== BTC seller
 
-_Coming soon._
+* Must click "Confirm payment" after having received it. Has till the end of the max. trade period to do so.
+* Must be available, in case of dispute, 48 hours after the max. trade period runs out.
+* Has 48 hours to respond to each arbitrator inquiry.
 
 == Arbitration
 
 When trading partners cannot successfully complete a trade on their own, the trade is disputed, and an arbitrator steps in to determine a solution.
```

Comment:
I would make the following section another big list, with the following introduction:

```
When trading partners cannot successfully complete a trade on their own, the trade is disputed, and an arbitrator steps in to determine a solution.

The following are possible circumstances and resulting outcomes.
```